webpack/webpack.common.js

```diff
@@ -0,0 +1,116 @@
+const path = require('path');
+const webpack = require('webpack');
+BASE_PATH = path.resolve(__dirname, '..')
+
+module.exports = {
+    entry: {
+        tests: {
+            import: './tests/main.js',
+        },
+        main: {
+            import: './src/main.js',
+            chunkLoading: false,
+        },
+    },
+    output: {
+        clean: true,
+        path: path.resolve(BASE_PATH, '_build'),
+    },
+    module: {
+        rules: [
+            {
+                test: /\.xml$/,
+                type: 'asset/source',
+            },
+            {
+                test: /templates\/.*\.html$/,
+                loader: path.resolve('src/template-loader.js'),
```

Comment:
Now that we've fixed the issues in tests, could this be replaced with `underscore-template-loader`?